tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/IAmazonSQSRequestWithQueueUrl.cs

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#nullable enable
```

Comment:
Any chance of null QueueUrl? 

FWIW, I typically find it doesn't require extra effort to mark most `string`s nullable, and it adds a certain extra sense of security 😄